src/clients/oAuth/refresh/refresh.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { RestClient } from '@src/utils/restClient';
+import { RefreshOAuth } from './types';
+import { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+import refresh from '.';
+
+jest.mock('@utils/restClient');
+
+describe('Testing OAuth, refresh', () => {
+	test('should make a POST request with the correct parameters', async () => {
+		const client = new MercadoPagoConfig({ accessToken: 'token', options: { timeout: 5000 } });
+		const oauthRequest: RefreshOAuth = {
```

Comment:
Inverteria o nome para manter um padrão com os outros. Ex.: OAuthRequest
```suggestion
		const oauthRequest: OAuthRefresh = {
```